File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int i,j=0;
        for (i = 0;i<nums.length;i++)
        {
            for(j=i+1;j<nums.length;j++)
            {
                if(nums[i] == nums[j])
                return true;
            }
        }
        return false;
 
    }
}

File: Data Structures & Algorithms/binary-search/submission-1.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:


        l = 0
        r = len(nums)-1
        if target not in nums:
            return -1
        while l<r:
            mid = l + (r-l)//2
            if target == nums[mid]:
                r = mid
            else:
                l = mid+1
            return l
        return -1
    